UIStepper Part 2:

alfimerino opened the pull request:
- Added constraints to view controller with stacks
- Removed extra labels

alfimerino committed:
UIStepper Part 2:
- Added constraints to view controller with stacks
- Removed extra labels

ivancr commented on BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Freddy/FreddyViewController.swift:
```diff
@@ -62,14 +56,14 @@ class FreddyViewController: UIViewController {
         textView4.dataDetectorTypes = UIDataDetectorTypes.phoneNumber
         
+        stackView.invalidateIntrinsicContentSize()
+        
```
Don't forget to delete this

ivancr commented on BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Freddy/FreddyViewController.swift:
```diff
@@ -62,14 +56,14 @@ class FreddyViewController: UIViewController {
         textView4.isEditable = false
         textView4.dataDetectorTypes = UIDataDetectorTypes.phoneNumber
         
+        stackView.invalidateIntrinsicContentSize()
```
```suggestion

```

alfimerino committed:
Formatting edits:
- Erased extra spaces between methods
- Removed  stackView.invalidateIntrinsicContentSize()

ivancr reviewed (approved):
Good job!
